src/components/main-page/FAQ/FAQ.tsx

```diff
@@ -0,0 +1,40 @@
+import { Heading, Section } from "components/ui";
+import { useTranslation } from "next-i18next";
+import styles from "./FAQ.module.scss";
+import { FAQDisclosure } from "./FAQDisclosure/FAQDisclosure";
+
+export type Question = {
+  question: string;
+  answer: string;
+};
+
+interface Section {
+  title: string;
+  questions: Question[];
+}
+
+const FAQ = () => {
```

Comment:
similarly as in comment below
```suggestion
export const FAQ = () => {
```